lib/core/base_wrapper.rb

```diff
@@ -8,9 +8,7 @@
 
 module BaseWrapper
   
-  Aws.config[:region] = 'us-east-1' # One-time configuration at module load.
-  
-  LOGGER = Logger.new($stdout, 'daily')
+  LOGGER = Logger.new($stderr, 'daily')
```

Comment:
@afred: The ssh-args script is intended to be run in backticks, so there we want to be sure that only the "real" output goes to STDOUT.